bayesian_cnn_prometheus/run_training_for_experiments.py

```diff
@@ -65,8 +64,6 @@ def create_config(weights_dir: str, overrides: Iterable[Override]):
 
 def create_experiment_dir(override: Iterable[Override]):
     experiment_name = "_".join(str(item) for item in override)
-    if not os.path.isdir(experiment_name):
-        os.mkdir(experiment_name)
```

Comment:
Wydaje mi się, że to nie jest potrzebne, bo tylko tworzy w repo folder np. stride_coś_tam, który jest pusty, a później i tak zapisuje rzeczy w folderze który jest tworzony w kolejnych linijkach

Reply:
Faktycznie. Stary kod.